coriolis/osmorphing/redhat.py

```diff
@@ -270,7 +270,7 @@ def _find_yum_repos(self, repos_to_enable=[]):
         installed_repos = []
         for file in repofiles:
             path = os.path.join(reposdir_path, file)
-            content = self._read_file(path).decode()
+            content = self._read_file_sudo(path).decode()
```

Comment:
Can you also please skip the file in case an error occurs?

Reply:
Updated the code now to send a `LOG.warn()` and to continue in case an error occurs while trying to read the repo file